pkg/generators/utils.go

```diff
@@ -0,0 +1,15 @@
+package generators
+
+func CombineMaps(a map[string]Generator, b map[string]Generator) map[string]Generator {
+	res := map[string]Generator{}
+
+	for k, v := range a {
+		res[k] = v
+	}
+
+	for k, v := range b {
+		res[k] = v
+	}
+
+	return res
+}
```

Comment:
I think we should return an error if there are duplicate keys detected (eg both generators happen to generate the same parameter name), unless the values are also the same. I think this behaviour is preferable versus using the reverse order of generators in the list to determine which duplicate key to use (which would be the unpatched  behaviour).